web_portal/server.py

```diff
@@ -0,0 +1,6 @@
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello, World!'
```

Comment:
Add home and about pages endpoints

Reply:
Will be added in the next tasks in the other branches